src/gluonts/model/predictor.py

```diff
@@ -580,16 +232,50 @@ def _worker_loop(
         output_queue.put((idx, worker_id, result))
 
 
+def GluonPredictor(*args, **kwargs):
+    from gluonts.mx.model.predictor import GluonPredictor as NewGluonPredictor
+
+    warnings.warn(
+        "gluonts.model.predictor.GluonPredictor is deprecated. Use gluonts.mx.model.predictor.GluonPredictor instead.",
+        DeprecationWarning,
+        stacklevel=2,
+    )
+    return NewGluonPredictor(*args, **kwargs)
+
+
+def SymbolBlockPredictor(*args, **kwargs):
+    from gluonts.mx.model.predictor import (
+        SymbolBlockPredictor as NewSymbolBlockPredictor,
+    )
+
+    warnings.warn(
+        "gluonts.model.predictor.SymbolBlockPredictor is deprecated. Use gluonts.mx.model.predictor.SymbolBlockPredictor instead.",
+        DeprecationWarning,
+        stacklevel=2,
+    )
+    return NewSymbolBlockPredictor(*args, **kwargs)
+
+
+def RepresentableBlockPredictor(*args, **kwargs):
+    from gluonts.mx.model.predictor import (
+        RepresentableBlockPredictor as NewRepresentableBlockPredictor,
+    )
+
+    warnings.warn(
+        "gluonts.model.predictor.RepresentableBlockPredictor is deprecated. Use gluonts.mx.model.predictor.RepresentableBlockPredictor instead.",
+        DeprecationWarning,
+        stacklevel=2,
+    )
+    return NewRepresentableBlockPredictor(*args, **kwargs)
+
+
```

Comment:
These should be removed since they're imported below?

```suggestion
```

Reply:
Yes thanks! Forget to remove that